src/haz3lweb/dune

```diff
@@ -93,7 +92,9 @@
    (flags (:standard)))))
 
 (rule
- (write-file js-of-ocaml-flags-dev "(:standard)"))
+ (write-file
+  js-of-ocaml-flags-dev
+  "(:standard --debug-info --no-inline --pretty --source-map)"))
```

Comment:
does this change the dev experience meaningfully?

Reply:
not afaict. but it does make the flags the same across all our build targets, and the same as david's setup, so at least it eliminates that potential source of confusion